### PR TITLE
HRK-2: Setting up prisma model and resolvers

### PR DESCRIPTION
**Pull request by fvalencia**

### CODE REVIEW

File: server/prisma/prisma.yml [hunk 1]
```diff
@@ -1,21 +1,30 @@
...
+    output: ../src/generated/schema.graphql
 
 seed:
   import: seed.graphql
 
 # Ensures Prisma client is re-generated after a datamodel change.
-hooks:
-  post-deploy:
-    - prisma generate
+# hooks:
```
Comment (yossely) [inline server/prisma/prisma.yml]: shouldn't this be uncommented? to generate again our prisma client automatically if we make a deploy

File: server/prisma/seed.graphql [hunk 2]
```diff
@@ -1,30 +1,114 @@
+# mutation {
```
Comment (yossely) [inline server/prisma/seed.graphql]: we don't need them, we're already prisma/graphql seniooor xD

File: server/src/resolvers/index.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+const Query = require('./query.js');
+const Mutation = require('./mutation.js');
+const Application = require('./application.js');
+const Candidate = require('./candidate.js');
+const Opening = require('./opening.js');
+const ApplicationStep = require('./application-step.js');
+const TemplateStep = require('./template-step.js');
+
+module.exports = {
```
Comment (yossely) [inline server/src/resolvers/index.js]: nice!

File: server/prisma/seed.graphql [hunk 4]
```diff
@@ -1,30 +1,114 @@
...
+          { id: 2, name: "English Interview" }
+          { id: 3, name: "Technical Test" }
+          { id: 4, name: "Team Lead Interview" }
+          { id: 5, name: "Client Interview" }
+        ]
+      }
     }
   ) {
     id
   }
```
Comment (lufeva) [inline server/prisma/seed.graphql]: Nice work on this one @yossely  and @fvalencia

**Pull request merged**